app/views/conversations/_conversation.haml

```diff
@@ -32,7 +32,7 @@
         .last_message
           - if conversation.messages.present?
             %em
-              = conversation.messages.last.text
+              = Diaspora::MessageRenderer.new(conversation.messages.last.text).markdownified
```

Comment:
`Conversation.find(2).messages.last.plain_text_without_markdown` if I use this it is throwing error.
and if I use ` Diaspora::MessageRenderer.new(Conversation.find(2).messages.last).plain_text_without_markdown` it is returning `#<Message:0x0000000ae7d3d0>`